packages/eslint-plugin/package.json

```diff
@@ -27,6 +28,18 @@
       "default": "./dist/configs/eslint-recommended-raw.js"
     }
   },
+  "typesVersions": {
+    "*": {
+      "use-at-your-own-risk/rules": [
+        "./rules.d.ts"
+      ],
+      "use-at-your-own-risk/eslint-recommended-raw": [
+        "./eslint-recommended-raw.d.ts"
+      ]
+    }
+  },
+  "types": "./index.d.ts",
+  "main": "./dist/index.js",
```

Comment:
We purposely do not list these keys. Please remove them.

Reply:
Done!